source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -52,14 +55,34 @@ public Paged<PimsAcquisitionFile> GetPage(AcquisitionFilter filter, HashSet<shor
                 throw new ArgumentException("Argument must have a valid filter", nameof(filter));
             }
 
+            IQueryable<PimsAcquisitionFile> query = GetCommonAquisitionFileQuery(filter, regions, filterPersonId);
+
+            var queryResult = query.ToList();
```

Comment:
nit: do we want to call ToList() here? That would bring the whole list into memory (instead of one page of results)...

The old code worked by doing the Skip, Take on the original query, and then calling ToList() - could we do the same here?

Reply:
updated